src/component/Nav/Modal/Modal.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import { css } from 'styled-components';
+
+const Modal = ({ onOffModal, name, children, id }) => {
+  const stop = e => {
+    e.stopPropagation();
+  };
+
+  const handlemodal = ({ target }) => {
+    onOffModal(target.id);
+  };
+  return (
+    <ModalCantainer name={name} id={id} onClick={handlemodal}>
+      <ModalBox name={name} onClick={stop}>
+        {children}
+      </ModalBox>
+    </ModalCantainer>
+  );
+};
+
+const ModalCantainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  z-index: 10000;
+  background: ${props => (props.name === 'write' ? '#00000080' : '')};
```

Comment:
```suggestion
  background: ${({ name }) => (name === 'write' ? '#00000080' : '')};
```
이 부분도 함수이기 때문에 인자 자리를 구조분해 할당 해줄 수 있습니다

Reply:
오 맞네요 ! 감사합니다 수정하겠습니다 !!